docker-compose.yaml

```diff
@@ -1,21 +1,53 @@
-version: "3"
+version: "3.9"
 
 services:
+  web:
+    depends_on:
+      - runner
+      - postgres
+      - grafana
+    build:
+      context: .
+      target: system
+    environment:
+      DATABASE_URL: postgres://postgres@postgres:5432/trento
+      EVENTSTORE_URL: postgres://postgres@postgres:5432/trento_eventstore
+      GRAFANA_PUBLIC_URL: http://grafana:3000
+      GRAFANA_API_URL: http://grafana:3000/api
+      RUNNER_URL: http://runner:8000
+    volumes:
+      - .:/app
+      - build_cache:/app/_build
+    ports:
+      - 4000:4000
+
+  runner:
```

Comment:
Usually the mocked runner adapter we have in the codebase is more than enough for me in development.

Locally we work with mocked data, that refer to some hosts that this runner might try to connect to in order to run checks, and it will fail.

What usage of the real runner container am I missing here? :thinking:
What would be the added value?

Reply:
I don't know how to use the mock runner and it's not documented anywhere, so this was the best thing I could come up with to get started with running the entire thing. 🙂 

Anyway this is not mutually exclusive with the mock, so I think we can keep both ways to run it, can't we?
If you think it's not that useful for your daily work, I will remove the service it from the new `mix start.full` task so that it doesn't bother you.

Reply:
Here's the mock runner 
https://github.com/trento-project/web/blob/main/lib/trento/application/integration/checks/adapter/mock_runner.ex
and it is enabled by default in `dev`.

> I think we can keep both ways to run it

Sure we could, I am trying to understand why, what's adding to DX.

The runner executes checks on targets, so unless we actually have some targets to hit, I miss other usages.
And if, for some reason there's the need to actually hit some targets, that means being working closer on that context, thus on the runner itself, so another service/repo.

Is the reason integration tests?
If so I believe that in the current scenario that kind of integration test is responsibility of the runner itself.
In relation to testing I believe we should consider contract testing here.

With these information, I am for removing it, and I'd be happy to change my mind if there's anything I am missing.